### ISSUE TITLE
Date errors during sync

### ISSUE THREAD

**Issue by develroo** [jellyfin/jellyfin-kodi#153]
When syncing TV shows this error sometimes appears, though no corresponding error happens on the server side. 

**To Reproduce**

Just begin fresh sync.
Watch the logs.

**Expected behavior**
Error free logs (Oh wouldn't that be nice? )

**Logs**
 ```
2019-12-07 21:39:10.644 T:140124612318976  NOTICE: JELLYFIN.helper.utils -> ERROR::resources/lib/helper/utils.py:484 date value out of range
                                            Traceback (most recent call last):
                                              File "resources/lib/helper/utils.py", line 480, in convert_to_local
                                                date = date.astimezone(tz.tzlocal())
                                              File "/home/user/.var/app/tv.kodi.Kodi/data/addons/script.module.dateutil/lib/dateutil/tz/_common.py", line 144, in fromutc
                                                return f(self, dt)
                                              File "/home/user/.var/app/tv.kodi.Kodi/data/addons/script.module.dateutil/lib/dateutil/tz/_common.py", line 258, in fromutc
                                                dt_wall = self._fromutc(dt)
                                              File "/home/user/.var/app/tv.kodi.Kodi/data/addons/script.module.dateutil/lib/dateutil/tz/_common.py", line 222, in _fromutc
                                                dtoff = dt.utcoffset()
                                              File "/home/user/.var/app/tv.kodi.Kodi/data/addons/script.module.dateutil/lib/dateutil/tz/tz.py", line 216, in utcoffset
                                                if self._isdst(dt):
                                              File "/home/user/.var/app/tv.kodi.Kodi/data/addons/script.module.dateutil/lib/dateutil/tz/tz.py", line 288, in _isdst
                                                if self.is_ambiguous(dt):
                                              File "/home/user/.var/app/tv.kodi.Kodi/data/addons/script.module.dateutil/lib/dateutil/tz/tz.py", line 250, in is_ambiguous
                                                (naive_dst != self._naive_is_dst(dt - self._dst_saved)))
                                            OverflowError: date value out of range
     2019-12-07 21:39:11.368 T:140124612318976  NOTICE: Previous line repeats 3 times.
````

**System (please complete the following information):**
 - OS: Debian
 - Jellyfin Version: 10.4.1
 - Kodi Version: 18.4 Git:20190906-nogitfound
 - Addon Version: 0.3.6

**Comment by mcarlton00** [comment]
Along the same lines, I've seen a few of these show up.
```
2019-12-08 17:16:14.551 T:139748538562304  NOTICE: JELLYFIN.helper.utils -> ERROR::resources/lib/helper/utils.py:484 year=1 is before 1900; the datetime strftime() methods require year >= 1900
                                            Traceback (most recent call last):
                                              File "resources/lib/helper/utils.py", line 482, in convert_to_local
                                                return date.strftime('%Y-%m-%dT%H:%M:%S')
                                            ValueError: year=1 is before 1900; the datetime strftime() methods require year >= 1900
```

Both errors don't seem to actual be real errors and prevent things from working, but it should still be looked into regardless

**Comment by mcarlton00** [comment]
The error I found is basically caused by bad data from the server.  We can hide the error, but it doesn't actually hurt anything.  Sometimes things that don't have properly formatted dates will get in.  For example, I just pulled this off an episode of a show:

`'PremiereDate': u'0001-01-01T00:00:00.0000000Z'`

Obviously this isn't a valid date and realistically shouldn't exist at all in the server, but there's not much to be done on our side.

@develroo I haven't managed to replicate the error that you've gotten.  Perhaps none of the media files in my library have malformed dates like what you're seeing.  The linked PR adds a little bit more detail to the error message so after the next release if it happens again we should be able to see what value it was trying to parse.
